assets/js/modules/analytics/components/dashboard/LegacyDashboardAcquisitionPieChart.js

```diff
@@ -90,11 +90,11 @@ const LegacyDashboardAcquisitionPieChart = ( { data, source } ) => {
 	return (
 		<div className="googlesitekit-chart googlesitekit-chart--pie">
 			<GoogleChart
+				chartType="PieChart"
 				data={ processedData }
+				loadingHeight={ 205 }
+				loadingWidth={ 205 }
```

Comment:
These should be strings:

```suggestion
				loadingHeight="205px"
				loadingWidth="205px"
```

Otherwise it throws errors:
![Screenshot from 2021-04-23 16-58-01](https://user-images.githubusercontent.com/873049/115882352-5a5aaf00-a455-11eb-8222-e4deeb911f8e.png)

Reply:
Whoops, thanks! 👍🏻